hypothesis-python/src/hypothesis/extra/ghostwriter.py

```diff
@@ -1221,7 +1227,7 @@ def maker(
         # that it's a good starting point to edit much of the rest.
         if identity is infer:
             try:
-                identity = find(operands, lambda x: True)
+                identity = find(operands, lambda x: True, settings=_quietly_settings)
```

Comment:
Is this related to the main change?

Reply:
...nope, I'd just written it when I saw the issue.  I'll move that to a new PR.